### ISSUE TITLE
Can't follow user from gist website

### ISSUE THREAD

**Issue by Maxim-Mazurok** [isaacs/github#1578]
When I'm on the stargazers page (https://gist.github.com/Maxim-Mazurok/404755f94091ab748f257309779c134b/stargazers) I can see the "Follow" button on one of my stargazers. But when I click on it, I'm getting 404 error on the page https://gist.github.com/users/follow?target=username (where `username` is the username of one I'd like to follow).
The workaround is to open his profile on GitHub like so: http://github.com/username (replacing `username` with an actual username that you're trying to follow).
The issue is reported to the GitHub support team.

**Comment by ShadAhm** [comment]
Also note that when you hover over the button it seems to display an incorrect username.

**Comment by Maxim-Mazurok** [comment]
No, the username seems fine to me, but the follow/unfollow issue is still there, getting 404
